client/webserver/site/src/css/icons.scss

```diff
@@ -1,9 +1,9 @@
 /* icons */
 
-[class^="ico-"],
-[class*=" ico-"] {
+[class^=ico-],
+[class*=ico-] {
```

Comment:
This change may not be exactly as intended, but I think in practice it will match what we want.

```
 3:9   ✖  Unexpected quotes around "ico-"   selector-attribute-quotes
 4:9   ✖  Unexpected quotes around " ico-"  selector-attribute-quotes
```

Alternatively we can modify the `selector-attribute-quotes` rule to go with `"always"` and then add double quotes to selector attributes in other places (or go with `null` to just disable this rule).  https://stylelint.io/user-guide/rules/list/selector-attribute-quotes/